presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2568,6 +2568,28 @@ public void testCreateBucketedTemporaryTable()
                 MaterializedResult.resultBuilder(SESSION, VARCHAR, VARCHAR).build());
     }
 
+    @Test
+    public void testCreateBucketedOnZeroColumnsTemporaryTable()
+            throws Exception
+    {
+        List<ColumnMetadata> columns = ImmutableList.<ColumnMetadata>builder()
```

Comment:
I think we can use `TEMPORARY_TABLE_COLUMNS` and `TEMPORARY_TABLE_DATA` ? :)

Reply:
Thanks, totally forgot to change it in this commit.